### DISCUSSION TITLE
Docker Zeiterfassung email don't work as expected...

### DISCUSSION THREAD

**Discussion by heikomaier** [kimai/kimai#5186] [General]
Hello, I pulled Docker images [kimai/kimai2:apache & mysql:8.3] and via docker compose everything works fine.
But email?
Following the kimai docs i could run
**./bin/console** mailer:test [a@b.de](mailto:a@b.de) --from=[kimai@b.de](mailto:kimai@b.de) **successfully!**
(there is only mailer:test and not mailer )

_You can test your email configuration by registering a new account or using the password reset function._
These 2 "things" don't work. MAILER_URL,MAILER_FROM and MAILER_DSN were set correctly.

So, may you help me about what's going wrong...
k.r. Heiko

**Comment by kevinpapst** [comment]
> (there is only mailer:test and not mailer )

The docs say `mailer:test`, not sure what you imply?

If the command works, then everything is fine. As the docs say.

The section https://www.kimai.org/documentation/emails.html#troubleshooting also tells you: 
<img width="1025" alt="Bildschirmfoto 2024-11-29 um 18 47 56" src="https://github.com/user-attachments/assets/340bf3be-15c4-48ac-9839-196c15b06077">

**Reply by heikomaier** [reply]
Hello Kevin,
thanks for your answer quickly.
Sry there is no email  by registering a new account.
Alsoi had no luck too with resetting the timer. Command is successfull.
In /opt/kimai/.env  I set MAILER_URL,MAILER_FROM . No emails.  And then I set  MAILER_DSN in **.env** too.
And ... **mailer:test** works, but **mailer:test only from console**.
Hope this helps. k.r. Heiko
